@kiva/kv-components/vue/KvSideSheet.vue

```diff
@@ -98,36 +118,86 @@ export default {
 			visible,
 			kvTrackFunction,
 			trackEventCategory,
+			expandEffect,
+			initialStyles,
 		} = toRefs(props);
 
 		const open = ref(false);
+		const animating = ref(false);
+
+		const modalStyles = ref({});
+
+		if (expandEffect.value) {
+			modalStyles.value = {
+				...initialStyles,
+				transition: 'none',
+			};
+		}
 
 		const closeSideSheet = () => {
 			open.value = false;
 			kvTrackFunction.value(trackEventCategory.value, 'click', 'side-sheet-closed');
+
+			if (expandEffect.value) {
+				animating.value = true;
+
+				modalStyles.value = {
```

Comment:
I'm not sure the specific CSS issue, but the header of the side sheet is currently disappearing before the animation starts, so the animation doesn't look smooth. The header of the side sheet should stay visible during the closing animation.